lms/djangoapps/course_api/blocks/tests/test_serializers.py

```diff
@@ -136,9 +138,6 @@ def create_serializer(self):
     def test_basic(self):
         serializer = self.create_serializer()
 
-        # verify root
-        self.assertEquals(serializer.data['root'], unicode(self.block_structure.root_block_usage_key))
```

Comment:
Removing assertions from tests makes me nervous :)

Can you explain why we no longer want this check?